frontend/src/libs/components/chat-layout/chat-layout.tsx

```diff
@@ -0,0 +1,13 @@
+import { ChatBody } from '#pages/chat/components/chat-body/chat-body.js';
+import { ChatHeader } from '#pages/chat/components/chat-header/chat-header.js';
```

Comment:
It's weird, that common component imports specialized page components, I think it shouldn't work that way. Also you have to create `pages/chat/components/components.ts` file to reexport all components from there